RedlinesProject/ControlCard.xaml.cs

```diff
@@ -39,6 +39,7 @@ public sealed partial class ControlCard : Page
             "CheckedNormal",
             "CheckedPointerOver",
             "CheckedPressed",
+            "Checked",
```

Comment:
For the problem you ran into (unable to add some selected states), I think it's just because we named the states differently.
https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/RadioButtons/RadioButtons_themeresources.xaml#L67

it should work if you add `PointerOverSelected` and `PressedSelected` here (I think).

Reply:
@kaiguo , unfortunately, it didn't work.